manage.py

```diff
@@ -4,6 +4,9 @@
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "pinry.settings.development")
+    sys.path.insert(0, './lib/python2.7/site-packages')
+
+    print(sys.path)
```

Comment:
This basically enables the virtualenv. I see two issues with it:

1. It has nothing to do with the vimeo integration.
2. It breaks if I choose to install the virtualenv anywhere else than in the root folder.